src/app/(main)/page.tsx

```diff
@@ -71,7 +81,7 @@ const Content = () => {
 
         <div className="bg-gray-100 p-6 rounded-lg mb-8 relative flex">
           <div className="flex-1">
-            {credentialContext?.hasStudentId ? (
+            {credentialContext?.hasStudentId =="true" ? (
```

Comment:
Is `hasStudentId` a string? We should probably fix that elsewhere rather than do a string comparison if so

Reply:
Is either "true" | "false"|"pending"